source/inplace_stop_token.cpp

```diff
@@ -27,7 +27,11 @@ inplace_stop_source::~inplace_stop_source() {
   assert((state_.load(std::memory_order_relaxed) & locked_flag) == 0);
 #ifndef NDEBUG
   for (auto* cb = callbacks_; cb != nullptr; cb = cb->next_) {
+#if __cpp_rtti
     printf("dangling inplace_stop_callback: %s\n", typeid(*cb).name());
+#else
+    printf("dangling inplace_stop_callback\n");
```

Comment:
This is a significantly less-useful error message than what you'd get with RTTI.  What do you think of having a debug-only member on the callback that uses `unifex::type_id` to ensure there's always a string available?